packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCardConnected.js

```diff
@@ -0,0 +1,57 @@
+import React, { memo } from 'react';
+import { Mutation } from 'react-apollo';
+import PropTypes from 'prop-types';
+
+import getCampusPrayerRequests from '../PrayerList/getPublicPrayerRequests';
+import flagPrayerRequest from './flagPrayerRequest';
+import PrayerCard from './PrayerCard';
+
+// eslint-disable-next-line react/display-name
+const PrayerCardConnected = memo(({ id, ...props }) => (
+  <Mutation
+    mutation={flagPrayerRequest}
+    update={async (cache, { data: { flagRequest } }) => {
+      const currentPrayerList = cache.readQuery({
+        query: getCampusPrayerRequests,
+      });
+      const flagIndex = currentPrayerList.getPublicPrayerRequests.findIndex(
+        (prayer) => prayer.id === flagRequest.id
+      );
+      const incrementFlag = () => {
+        currentPrayerList.getPublicPrayerRequests[flagIndex].flagCount += 1;
+      };
+      const updatedPrayerList = () => {
```

Comment:
On line 29 you call this function and pass it `(currentPrayerList.getPublicPrayerRequests)`. Do you not need to specify that as a prop to this function? More a question than a critique.